openbb_platform/providers/fmp/openbb_fmp/utils/helpers.py

```diff
@@ -144,3 +149,27 @@ def get_interval(value: str) -> str:
     }
 
     return f"{value[:-1]}{intervals[value[-1]]}"
+
+
+# some fmp endpoint return date in EST without a timezone, this function will parse it
```

Comment:
They actually return the date in the timezone of the exchange, without a timezone. For example: 

https://financialmodelingprep.com/api/v3/historical-chart/5min/MNDI.L?
```
  {
    "date": "2024-04-08 08:05:00",
    "open": 1400.5,
    "low": 1396.5,
    "high": 1400.5,
    "close": 1398,
    "volume": 5298
  },
  {
    "date": "2024-04-08 08:00:00",
    "open": 1395,
    "low": 1395,
    "high": 1407.5,
    "close": 1399,
    "volume": 4883
  }
```

The London Stock Exchange is open Monday through Friday from 8:00 am to 4:30 pm British Summer Time (GMT+01:00).

You are localizing that as 8:00 AM EST, which is not a correct TZ conversion.

Reply:
smh they really are the best 💀

Reply:
This is exactly the reason why it's not localized, it's better to leave it alone than to make it incorrectly tz-aware.

Reply:
then I need to figure out the current timezone per symbol, or else all dates will be treated as UTC on the frontend. RIP

Reply:
Seems like something frontend needs to deal with independently of the fetcher. These should stay tz-unaware from the fetcher.

Reply:
@deeleeramone if we have to do this on FE can you get us a list of all exchanges and the time zone they in

 Ty

Reply:
Not from FMP directly, at least AFAICT.